src/app/ToDo/state/todo.effects.ts

```diff
@@ -3,44 +3,27 @@ import { Actions, createEffect, ofType } from '@ngrx/effects';
 import { Action } from '@ngrx/store';
 import { Observable, of } from 'rxjs';
 import { catchError, map, mergeMap } from 'rxjs/operators';
+import { ToDo } from '.';
 import * as ToDoActions from './todo.action';
 import { ToDoHttpService } from './todo.httpservice';
-import ToDo from './todo.model';
 
 @Injectable()
 export class ToDoEffects {
-  constructor(private todoService: ToDoHttpService, private action$: Actions) {}
+    constructor(private todoService: ToDoHttpService, private action$: Actions) { }
 
-  GetToDos$: Observable<Action> = createEffect(() =>
-    this.action$.pipe(
-      ofType(ToDoActions.BeginGetToDoAction),
-      mergeMap(action =>
-        this.todoService.getToDos().pipe(
-          map((data: ToDo[]) => {
-            return ToDoActions.SuccessGetToDoAction({ payload: data });
-          }),
-          catchError((error: Error) => {
-            return of(ToDoActions.ErrorToDoAction(error));
-          })
+    GetToDos$: Observable<Action> = createEffect(() =>
+        this.action$.pipe(
+            ofType(ToDoActions.LoadTodos),
+            mergeMap(() =>
```

Comment:
Why did you use the `mergeMap` method?

I think `switchMap` is enough in this case.